src/highlight/HighlightList.tsx

```diff
@@ -92,4 +92,6 @@ export function HighlightList({ activeId = null, annotations, className, onSelec
     );
 }
 
+export { HighlightList as HighlightListComponent };
```

Comment:
Seems like this is the same thing as exporting the function inline, other than the renaming? The importing class should probably manage that.

Reply:
I had done it for some mocking to work, let me see if I can revert that